src/mlpack/prereqs.hpp

```diff
@@ -72,7 +72,6 @@ using enable_if_t = typename enable_if<B, T>::type;
 // defined, but we still need to define it (as nothing) so that the mlpack
 // serialization shim compiles.
 #include <boost/serialization/serialization.hpp>
-#include <boost/serialization/vector.hpp>
```

Comment:
It's fine to remove this from here, but I think it would be useful to add a comment pointing out why we are including `vector.hpp` somewhere else.